app/controllers/close_contacts_controller.rb

```diff
@@ -42,4 +42,15 @@ def update
                                created_by: current_user.email,
                                comment: "User edited a close contact (ID: #{cc.id}).")
   end
+
+  # Delete an existing close contact record
+  def destroy
+    redirect_to root_url unless current_user.can_create_patient_close_contacts?
+    cc = current_user.get_patient(params.permit(:patient_id)[:patient_id])&.close_contacts.find_by(id: params.permit(:id)[:id])
+    cc.update(archived: true)
+    cc.save
+    History.close_contact_edit(patient: params.permit(:patient_id)[:patient_id],
+                              created_by: current_user.email,
```

Comment:
there are probably multiple rubocop errors that need to be fixed, one of which is the indentation here, it'll also tell you to add blank lines after guard clauses, make sure to always check rubocop